monday/query_joins.py

```diff
@@ -381,24 +460,71 @@ def get_boards_by_id_query(board_ids):
     }''' % board_ids
 
 
-def get_columns_by_board_query(board_ids):
-    return '''query
-        {
-            boards(ids: %s) {
+def duplicate_board_query(
+    board_id: int,
+    duplicate_type: DuplicateTypes,
+    board_name: str = None,
+    workspace_id: int = None,
+    folder_id: int = None,
+    keep_subscribers: bool = None,
+) -> str:
+    board_name = board_name if board_name else ""
+    workspace_id = workspace_id if workspace_id else None
+    folder_id = folder_id if folder_id else None
+    keep_subscribers = keep_subscribers if keep_subscribers else False
+
+    params = """board_id: %s, duplicate_type: %s, board_name: \"%s\"""" % (
```

Comment:
[very non-blocking] Why does no one in this repo use f-strings?!

[edit]  And then, randomly, later on, they do!!

Reply:
The non f-strings were almost certainly my fault when I first wrote this, what feels like eons ago, and then f-strings were added later by either myself or contributors. I've been meaning to go through and clean this up.